### ISSUE TITLE
`no-warning-comments`: Support JSDoc "todo" check with option to disable

### ISSUE THREAD

**Issue by brettz9** [eslint/eslint#11056]
**What rule do you want to change?**

`no-warning-comments`

**Does this change cause the rule to produce more or fewer warnings?**

More.

**How will the change be implemented? (New option, new default behavior, etc.)?**

Preferably as a default but with an option to disable this new check (since JSDoc to-dos might be used for more formal long-term to-dos whereas inline to-do comments could be during debugging).

**Please provide some example code that this change will affect:**

```js
/**
* @todo Should have way to flag this too
*/
```

**What does the rule currently do for this code?**

Nothing.

**What will the rule do after it's changed?**

Flag it.

**Are you willing to submit a pull request to implement this change?**

Not at this time.

**Comment by nzakas** [comment]
Thanks for the suggestion. As of yesterday, we have decided to officially [end-of-life JSDoc support](https://eslint.org/blog/2018/11/jsdoc-end-of-life) in ESLint. All JSDoc features are now deprecated and we won't be fixing bugs or making any further improvements to those features.

We are recommending that people transition over to use the [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc) plugin instead of the built-in rules in ESLint.

Thanks for understanding and we apologize for any inconvenience.
